substrate/primitives/keyring/src/ethereum.rs

```diff
@@ -0,0 +1,257 @@
+//   {
+//     name: 'Alith',
+//     p: '0x02509540919faacf9ab52146c9aa40db68172d83777250b28e4679176e49ccdd9f',
+//     s: '0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133', // nosemgrep
+//     type: 'ethereum'
+//   },
+//   {
+//     name: 'Baltathar',
+//     p: '0x033bc19e36ff1673910575b6727a974a9abd80c9a875d41ab3e2648dbfb9e4b518',
+//     s: '0x8075991ce870b93a8870eca0c0f91913d12f47948ca0fd25b49c6fa7cdbeee8b', // nosemgrep
+//     type: 'ethereum'
+//   },
+//   {
+//     name: 'Charleth',
+//     p: '0x0234637bdc0e89b5d46543bcbf8edff329d2702bc995e27e9af4b1ba009a3c2a5e',
+//     s: '0x0b6e18cafb6ed99687ec547bd28139cafdd2bffe70e6b688025de6b445aa5c5b', // nosemgrep
+//     type: 'ethereum'
+//   },
+//   {
+//     name: 'Dorothy',
+//     p: '0x02a00d60b2b408c2a14c5d70cdd2c205db8985ef737a7e55ad20ea32cc9e7c417c',
+//     s: '0x39539ab1876910bbf3a223d84a29e28f1cb4e2e456503e7e91ed39b2e7223d68', // nosemgrep
+//     type: 'ethereum'
+//   },
+//   {
+//     name: 'Ethan',
+//     p: '0x025cdc005b752651cd3f728fb9192182acb3a9c89e19072cbd5b03f3ee1f1b3ffa',
+//     s: '0x7dce9bc8babb68fec1409be38c8e1a52650206a7ed90ff956ae8a6d15eeaaef4', // nosemgrep
+//     type: 'ethereum'
+//   },
+//   {
+//     name: 'Faith',
+//     p: '0x037964b6c9d546da4646ada28a99e34acaa1d14e7aba861a9055f9bd200c8abf74',
+//     s: '0xb9d2ea9a615f3165812e8d44de0d24da9bbd164b65c4f0573e1ce2c8dbd9c8df', // nosemgrep
+//     type: 'ethereum'
+//   }
```

Comment:
:rocket:

Reply:
Sudo account is now "//Alith". Do you need it to be something else?
https://github.com/mangata-finance/mangata-node/blob/5f7db1920931022da15e7248b0b180603852f8b1/rollup/node/src/chain_spec.rs#L84

Reply:
all good!